hnn_core/tests/test_dipole.py

```diff
@@ -44,6 +44,28 @@ def test_dipole(tmpdir):
                        "average of 2 trials"):
         dipole_avg = average_dipoles([dipole_avg, dipole_read])
 
+    # test postproc
+    hnn_core_root = op.dirname(hnn_core.__file__)
+    params_fname = op.join(hnn_core_root, 'param', 'default.json')
+    params = read_params(params_fname)
+    params_reduced = params.copy()
+    params_reduced.update({'N_pyr_x': 3,
+                           'N_pyr_y': 3,
+                           'tstop': 25,
+                           't_evprox_1': 5,
+                           't_evdist_1': 10,
+                           't_evprox_2': 20,
+                           'N_trials': 2})
+    net = Network(params_reduced)
+    dpls_raw = simulate_dipole(net, postproc=False)
+    dpls = simulate_dipole(net, postproc=True)
```

Comment:
If we need to run simulations for this unit test, would it be best to put it in test_parallel_backends.py?